src/realm-roles/RolesList.tsx

```diff
@@ -69,7 +75,9 @@ export const RolesList = ({
 
   const RoleDetailLink = (role: RoleRepresentation) => (
     <>
-      <RoleLink role={role} />
+      <RoleLink role={role}>
+        <>{role.name}</>
```

Comment:
Why do we need a fragment here? I believe you can render the children directly.
```suggestion
        {role.name}
```